resources/views/layout/header_mobile/user.blade.php

```diff
@@ -6,7 +6,7 @@
     @if (Auth::check())
         <div
             class="navbar-mobile-item__main js-react--user-card"
-            data-user="{{ json_encode(Auth::user()->defaultJson()) }}"
+            data-user="{{ json_encode(json_item(auth()->user(), 'UserCompact', ['cover', 'country', 'groups'])) }}"
```

Comment:
how about add mount option like `data-user-current="1"` and then check that in props function in `register-components` :thinking: 

Also I noticed the one for `user-card-store` doesn't even need to pass the user at all (also it's missing group lol).

<sub>and then there's unused `objects._usercard` view...</sub>